cmd/cloud-controller-manager/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+- thockin
+- luxas
+- wlan0
+- andrewsykim
```

Comment:
At this point I think it makes sense to put cheftako as I will be maintaining and doing a lot of the dev on this.

Reply:
Added you to the root owners and remove this file.